torchgen/gen.py

```diff
@@ -2076,12 +2086,38 @@ def gen_backend_select() -> Dict[str, List[str]]:
     schema_selector = selector
     if force_schema_registration:
         schema_selector = SelectiveBuilder.get_nop_selector()
+    schema_static_template = CodeTemplate(
+        """
+TORCH_LIBRARY($ns, m) {
+    $schema_registrations_body
+};"""
+    )
+    ns_native_functions: Dict[str, List[NativeFunction]] = defaultdict(list)
+    for native_function in native_functions:
+        ns_native_functions[native_function.namespace].append(native_function)
+    schema_registrations = ""
+    aten_schema_registrations = []
+    for namespace, functions in ns_native_functions.items():
+
+        schema_registrations_body = list(
+            mapMaybe(RegisterSchema(schema_selector), functions)
+        )
+        if namespace == "aten":
+            aten_schema_registrations = schema_registrations_body
```

Comment:
I think this deserves a comment - you're treating "aten` specially, because you don't want to generate the `TORCH_LIBRARY` call for it (because the C++ template file already has it)